tests/data/test_dataset_generator.py

```diff
@@ -7,48 +7,70 @@
 import pytest
 from pytest_mock import MockerFixture
 
-from coin_test.data import ReturnsDatasetGenerator
+from coin_test.data import ReturnsDatasetGenerator, StitchedChunkDatasetGenerator
 from coin_test.data.metadata import MetaData
 from coin_test.util import AssetPair, Ticker
 
 
-def test_dataset_generator_initialized(hour_data_indexed_df: pd.DataFrame) -> None:
-    """Initialize the ResultDatasetGenerator."""
+def test_chunked_dataset_generator_initialized(
+    hour_data_indexed_df: pd.DataFrame,
+) -> None:
+    """Initialize the StitchedChunkDatasetGenerator."""
     mock_dataset = Mock()
     pair = AssetPair(Ticker("BTC"), Ticker("USDT"))
     metadata = MetaData(pair, freq="H")
     mock_dataset.metadata = metadata
     mock_dataset.df = hour_data_indexed_df
+    chunk_size = 5
```

Comment:
Can we test with one? I thought there was some special case that probably needs to be tested -- might have missed it